docs/specifiers.rst

```diff
@@ -28,6 +28,10 @@ Usage
     >>> combined_spec &= "!=1.1"
     >>> combined_spec
     <SpecifierSet('!=1.1,>=1.0,~=1.0')>
+    >>> # We can iterate over the SpecifierSet to recover the
+    >>> # individual specifiers
+    >>> sorted(combined_spec, key=lambda x: x.operator)
```

Comment:
```suggestion
    >>> sorted(combined_spec, key=str)
```

Reply:
While key=str is a more minimal change to make doctests pass, it's also perhaps a little more inscrutable as a reader. Accessing `.operator` shows off some more of the API, so is maybe also more real-world-y.
(I don't feel strongly, so feel free to apply the change if you don't agree)

Reply:
Sorting by operator does not really make sense though. Personally I prefer `str` here.

Reply:
If you really really want to show off the API, sorting by version is probably more meaningful (but the syntax could be a bit too verbose for a short example).

Reply:
Great, made the change!